docs/validators/node-configuration.md

```diff
@@ -0,0 +1,84 @@
+- `external.toml` holds external drivers' configuration (for example: Elasticsearch or event notifier)                                                                                       
+- `gasSchedules` is the directory that contains all the gas schedules to be used for SC execution, depending on activation epochs specified on external.toml->GasSchedule->GasScheduleByEpochs 
+- `genesisContracts` is the directory that contains the WASM contracts that were deployed at Genesis time                                                                                                
+- `genesis.json`  contains all the addresses and their balance/active delegation at Genesis time                                                                                             
+- `genesisSmartContracts.json` specifies the SCs to be deployed at Genesis time, alongside additional parameters                                                                                           
+- `nodesSetup.json` holds all the Genesis nodes public keys, alongside their wallet address                                                                                                    
+- `p2p.toml` holds peer-to-peer configurable values, such as the number of peers to connect to                                                                                          
+- `prefs.toml` holds a set of custom configuration values, that should not be replaced from an upgrade to another                                                                         
+- `ratings.toml` holds the parameters used for the nodes' rating mechanism, for example the start rating, decrease steps, and so on                                                         
+- `systemSmartContractsConfig.toml` holds configuration values for System Smart Contracts, such as the Staking, ESDT, or Governance                                                                            
+
+### Overloading config.toml values 
+
+As mentioned in the above descriptions, `prefs.toml` should not be overwritten when performing an upgrade. 
+
+However, there are some custom values that nodes operators use (antiflood disabled or with fewer constraints, db lookup extension, and so on) 
+and they don't want these values to be changed during an upgrade. 
+
+For this use-case, release `v1.4.x` introduces the `OverridableConfigTomlValues` setting inside `prefs.toml` that is able to override certain configuration
+values from `config.toml`.
+
+Here's how to use it:
+
+```
+   OverridableConfigTomlValues = [
+     { Path = "StoragePruning.NumEpochsToKeep", Value = "4" },
+     { Path = "MiniBlocksStorage.Cache.Name", Value = "MiniBlocksStorage" }
+   ]
+```
+
+Therefore, after each upgrade, the node will override these values to the new provided values. The path should be observed
+in `config.toml` file before setting a new overridable value.
```

Comment:
`The path points to an entry in config.toml` (or similar)

Reply:
fixed